### ISSUE TITLE
Anyway to add a list of MD5's to downloads?

### ISSUE THREAD

**Issue by GatoExtremity** [Bionus/imgbrd-grabber#1598]
I've got a list of over 12 thousand MD5s (and nothing else; no ID, tags, or anything) from e621 that I'd like to download through the software, but I don't see, or at least I'm missing, a way to add all of these to the downloads without requiring me to individually copy-paste every single MD5 into the single images list..

Is there any way to add all of these MD5s into the download list at once, or do I actually have to suffer through manually adding each and every one of those individually?

**Comment by Bionus** [comment]
Yes, see issue #1480.
It's currently in the [nightly version](https://github.com/Bionus/imgbrd-grabber/releases/tag/nightly) and will land in 7.0.0 when I'm done with the finishing touches.

⚠️ This feature has never been tried with huge numbers like 12K images, so I would suggest to try smaller amounts first (a few hundreds / 1K) to make sure it can handle the load first (especially in RAM).

However, how did you get the 12K MD5s on your end? Maybe there's a better way to start those downloads (if they're already on your disk and want to re-tag them for example, or if they're flagged as your favorites on some website, there are built-in solutions).

![image](https://user-images.githubusercontent.com/882719/47619656-bd1b9b80-dae1-11e8-928a-2a053ec47854.png)

![image](https://user-images.githubusercontent.com/882719/47619663-c73d9a00-dae1-11e8-911c-8a56a029de05.png)

_(in this example for IDs, but it works for MD5s as well)_

**Comment by GatoExtremity** [comment]
Whew, that's a relief..

About where I got those, well.. I've only recently realized I've been manually saving samples from e621 instead of the original images this entire time. (good job me)
So I figured I'd get the program and use it to re-download my entire collection in full quality by using the list of the image names, which I've kept as their md5's. This should help.
Thank you for your time!
